snapcraft.yaml

```diff
@@ -915,13 +931,22 @@ parts:
       - lib/*/libnuma*so*
       - lib/*/libpixman*so*
       - lib/*/libusbredir*so*
+      - lib/*/libfdt*.so*
       - share/qemu/keymaps*
-      - share/qemu/efi-virtio.rom*
-      - share/qemu/kvmvapic.bin*
-      - share/qemu/s390-*.img*
-      - share/qemu/slof.bin*
-      - share/qemu/vgabios-*.bin*
       - share/qemu/bios-256k.bin*
+      - share/qemu/efi-virtio*.rom
```

Comment:
There is only one `efi-virtio` ROM file so I'd use:
```suggestion
      - share/qemu/efi-virtio.rom
```

Reply:
this is to make them option on the non-qemu archs.

Reply:
I think i could further improve the build to not install these packages either on those archs.

Reply:
yeah and it didnt work with the prime section either when i tried it.

Reply:
ill go with the optional indicator instead for prime files